tests/SocketTest.cpp

```diff
@@ -2,14 +2,69 @@
 #define CATCH_CONFIG_MAIN
 #include "catch.hpp"
 
-#include "pony/Address.hpp"
-#include "pony/Socket.hpp"
+#include <pony/Socket.hpp>
+
+#include <ostream>
 
 using namespace pony;
 
-TEST_CASE("Socket dummy test", "[]")
+TEST_CASE("Invalid initialization", "[Socket]")
 {
+    SocketsInit();
+
     Socket socket;
 
     REQUIRE(false == socket.IsOpen());
+
+    SocketsKill();
+}
+
+const uint16_t port = 33333;
+const uint32_t localhost = (0x1 | 127 << 24);
+const int8_t data[] = "Hello world!";
```

Comment:
why `int8_t` when it's clearly a `char` array?

Also why is it here in global scope and not in the test case's scope?

Reply:
ok